tools/run_block.py

```diff
@@ -93,17 +92,10 @@ def run_generator(
     block_generator: BlockGenerator, constants: ConsensusConstants, max_cost: int, height: uint32
 ) -> List[CAT]:
 
-    if height >= DEFAULT_CONSTANTS.SOFT_FORK_HEIGHT:
-        # conditions must use integers in canonical encoding (i.e. no redundant
-        # leading zeros)
-        # the division operator may not be used with negative operands
-        flags = COND_CANON_INTS | NO_NEG_DIV
-    else:
-        flags = 0
-
-    _, result = block_generator.program.run_with_cost(max_cost, flags, block_generator.generator_refs)
```

Comment:
as far as I can see, the problem is that `generator_refs` need to be wrapped in a list here

Reply:
Yes, thanks!